docs/INSTALLATION_LOCAL_BENGALI.md

```diff
@@ -0,0 +1,85 @@
+# স্থানীয় মেশিন ইনস্টল এবং চালান
+
+### 1. আপনার ডেস্কটপে ক্লোন chat.susi.ai সংগ্রহস্থল
+
+`git clone https://github.com/fossasia/chat.susi.ai`
+
+![git clone](./img/gitclone.png)
+
+### 2. ডাইরেক্টরিটি ক্লোনড রিপোজিটরিতে পরিবর্তন করুন
+
+`cd chat.susi.ai`
+
+![cd](./img/cd.png)
+
+### 3. ইনস্টলেশনের নির্ভরতা
+
+`npm install`
+
+![npm install](./img/npmistall.png)
+
+আউটপুট: * (নমুনা) *
+
+
+    npm WARN deprecated tar.gz@0.1.1: â ï¸  WARNING â ï¸ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar
```

Comment:
@arnav-mandal1234 Can you please explain why are these required?

Reply:
@rishiraj824 There was an issue with adding more language and I have translated the English Docs to Bengali.

Reply:
```
npm WARN deprecated tar.gz@0.1.1: â ï¸  WARNING â ï¸ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar
```
Ok, but this text comes in the between the code. I think we can remove this.

Reply:
Its the part of the code Please follow the English instruction carefully.